ompi/mpi/man/man3/MPI_Comm_spawn_multiple.md

```diff
@@ -0,0 +1,257 @@
+```
+
+
+# Input Parameters
+
+* `count` : Number of commands (positive integer, significant to MPI only at
+*root* -- see NOTES).
+* `array_of_commands` : Programs to be executed (array of strings, significant only at
+*root*).
+* `array_of_argv` : Arguments for *commands* (array of array of strings, significant
+only at *root*).
+* `array_of_maxprocs` : Maximum number of processes to start for each command (array of
+integers, significant only at *root*).
+* `array_of_info` : Info objects telling the runtime system where and how to start
+processes (array of handles, significant only at *root*).
+* `root` : Rank of process in which previous arguments are examined (integer).
+* `comm` : Intracommunicator containing group of spawning processes (handle).
+
+# Output Parameters
+
+* `intercomm` : Intercommunicator between original group and the newly spawned group
+(handle).
+* `array_of_errcodes` : One code per process (array of integers).
+* `IERROR` : Fortran only: Error status (integer).
+
+# Description
+
+`MPI_Comm_spawn_multiple` is identical to `MPI_Comm_spawn(3`) except that it
+can specify multiple executables. The first argument, count, indicates
+the number of executables. The next three arguments are arrays of the
+corresponding arguments in `MPI_Comm_spawn(3)`. The next argument,
+`array_of_info`, is an array of `info` arguments, one for each
+executable. See the INFO ARGUMENTS section for more information.
+For the Fortran version of `array_of_argv`, the element
+`array_of_argv(i,j)` is the jth argument to command number i.
+
+In any language, an application may use the constant `MPI_ARGVS_NULL`
+(which is likely to be (char )0 in C) to specify that no arguments
+should be passed to any commands. The effect of setting individual
+elements of `array_of_argv` to `MPI_ARGV_NULL` is not defined. To specify
+arguments for some commands but not others, the commands without
+arguments should have a corresponding `argv` whose first element is null
+((char )0 in C and empty string in Fortran).
+
+All of the spawned processes have the same `MPI_COMM_WORLD`. Their ranks
+in `MPI_COMM_WORLD` correspond directly to the order in which the commands
+are specified in `MPI_Comm_spawn_multiple`. Assume that m1 processes are
+generated by the first command, m2 by the second, etc. The processes
+corresponding to the first command have ranks 0, 1,..., m1-1. The
+processes in the second command have ranks m1, m1+1, ..., m1+m2-1. The
+processes in the third have ranks m1+m2, m1+m2+1, ..., m1+m2+m3-1, etc.
+
+The `array_of_errcodes` argument is 1-dimensional array of size
+         _ count
+        \      n ,
+        /_ i=1   i
+where i is the ith element of `array_of_maxprocs`. Command number i
+corresponds to the i contiguous slots in this array from element
+                        _              _
+         _ i-1         \|   _ i          \|
+        \      n ,  to \|  \      n      \| -1
+        /_ j=1  i      \|_ /_ j=1  j    \_\|
```

Comment:
I went in and manually added the \ because my text editor was syntax highlighting the bars... I will go in and remove those, the only potential alternative I can think of is making an image and referencing it, but that wouldn't work in a terminal

Reply:
Maybe put them in a verbatim section...?